src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -280,6 +282,16 @@ void IStreamsExecutor::Config::SetConfig(const std::string& key, const std::stri
                        << ". Expected only non negative numbers";
         }
         _small_core_offset = val_i;
+    } else if (key == CONFIG_KEY_INTERNAL(ENABLE_HYPER_THREAD)) {
+        int val_i;
+        if (value == CONFIG_VALUE_INTERNAL(YES)) {
+            val_i = true;
```

Comment:
why not directly assign value to _enable_hyper_thread

Reply:
Ok, I will remove the redundant code.

Reply:
done